projects/ad9081_fmca_ebz/common/ad9081_fmc.txt

```diff
@@ -75,8 +77,8 @@ G13       LA08_N         IRQB<1>            irqb[1]             LVCMOS15    #N/A
 H17       LA11_N         HMC_GPIO1          hmc_gpio1           LVCMOS15    #N/A
 
 D11       LA05_P         SPI0_CSB           spi0_csb            LVCMOS15    #N/A   
-H10       LA04_N         SPI0_SCLK          spi0_sclk           LVCMOS15    #N/A
-H11       LA04_P         SPI0_MOSI          spi0_mosi           LVCMOS15    #N/A
+H11       LA04_N         SPI0_SCLK          spi0_sclk           LVCMOS15    #N/A
+H10       LA04_P         SPI0_MOSI          spi0_mosi           LVCMOS15    #N/A
```

Comment:
Can we change the order to have P then N, similar with other pairs?

Reply:
done